docs/tutorials/hitl.rst

```diff
@@ -201,20 +201,18 @@ In the ``suggest_and_generate_image`` function, a new Trial is obtained and new
 .. code-block:: python
     :linenos:
 
-    def start_optimization(
-        storage: optuna.storages.BaseStorage, artifact_backend: FileSystemBackend
-    ) -> NoReturn:
+    def start_optimization(artifact_backend: FileSystemBackend) -> NoReturn:
         # 1. Create Study
-        sampler = optuna.samplers.TPESampler(constant_liar=True)
+        storage = "sqlite:///db.sqlite3"
```

Comment:
[nits] It seems that we do not need this line.

Reply:
Correct! Good catch.

```suggestion
```